pkg/jobrunaggregator/jobrunbigqueryloader/test_run_uploader.go

```diff
@@ -51,13 +49,34 @@ func (o *testRunUploader) uploadTestSuite(ctx context.Context, jobRun jobrunaggr
 		}
 	}
 
-	toInsert := []bigquery.ValueSaver{}
+	toInsert := []*jobrunaggregatorapi.TestRunRow{}
 	for i := range suite.TestCases {
 		testCase := suite.TestCases[i]
 		if testCase.SkipMessage != nil {
 			continue
 		}
-		toInsert = append(toInsert, newTestRunRow(jobRun, prowJob, currSuites, testCase))
+
+		var status string
+		switch {
+		case testCase.FailureOutput != nil:
+			status = "Failed"
+		case testCase.SkipMessage != nil:
+			status = "Skipped"
+		default:
+			status = "Passed"
+		}
+
+		var testSuiteStr string
+		testSuiteStr = suite.Name
```

Comment:
it's the test suite logically

`testSuiteStr := strings.Join(currSuites, "|||")` ?

This looks like it would assign the current test case the value of all the child suites.